### PR TITLE
Add link purpose

### PR DESCRIPTION
**Pull request by ronaldmansveld**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: test/accessibility/tests/unauthenticatedUrls.js [hunk 1]
```diff
@@ -25,13 +25,90 @@ describe('Pa11y tests', function () {
...
+        })
+        await MockApp.startServer();
+        const actions = regitrationActions();
+        actions.push(...PallyActions.clickElement('app-check-your-answers button'));
+        actions.push(...PallyActions.waitForPageWithCssLocator('.govuk-error-summary'));
+        await pa11ytest(this, actions);
+    });
+
+
+    function regitrationActions(){
```
Comment (paulhowes-HMCTS) [inline test/accessibility/tests/unauthenticatedUrls.js:68]: Typo - I assume this should be registrationActions

File: test/e2e/features/pageObjects/mailinatorService.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ class MailinatorService{
         this.currentInboxLabel = "#inbox_page_title .title_left:nth-of-type(3) .top_search";
         this.latestEmailRowCssSelector = ".x_panel .x_content tbody tr:nth-of-type(1) td:nth-of-type(3)"
         this.activationEmaillinkCSsSelector = ".content tr:nth-of-type(2) td:nth-of-type(2) a:nth-of-type(1)";
-
+//a[contains(text(),'Confirm your email address')]
```
Comment (paulhowes-HMCTS) [inline test/e2e/features/pageObjects/mailinatorService.js:10]: Being finicky I'd say commented out code should be removed if no longer required.

File: test/accessibility/tests/unauthenticatedUrls.js (hunk 1 above)
Comment (paulhowes-HMCTS) [inline test/accessibility/tests/unauthenticatedUrls.js:68]: Typo - I assume this should be registrationActions

File: package.json [hunk 3]
```diff
@@ -61,7 +61,7 @@
     "@hmcts/frontend": "0.0.41-alpha",
     "@hmcts/nodejs-healthcheck": "^1.6.0",
     "@hmcts/properties-volume": "0.0.9",
-    "@hmcts/rpx-xui-common-lib": "0.1.19",
+    "@hmcts/rpx-xui-common-lib": "0.1.36",
```
Comment (paulhowes-HMCTS) [inline package.json:64]: Current version in master is 0.1.48 - should this include ^?

**Review by paulhowes-HMCTS — CHANGES_REQUESTED** [review]
A couple of finicky queries and a query on the version number of the rpx-xui-common-lib need looking at.

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@ronaldmansveld - just the 1 comment that needs reviewing; otherwise looks OK.

**Pull request merged**